README.md

```diff
@@ -197,7 +197,24 @@ disabledSlider.disable(true);
 
 > #### Slider.prototype.disable
 > 
-> Makes the slider handle unmovable
+> Disabled makes the slider handle unmovable
+> Pass true to disable the slider
+> Pass false to reenable the slider
+> Defaults to true
+
+### Destory the slider
+
+```javascript
+var memoryLeak = new Slider(document.getElementById('leakingSlider'));
+memoryLeak.destroy(true);
```

Comment:
Non-blocking question, @chrisrng , can we expand the `.destroy` accepted parameter to accept as object instead of a boolean flag? Perhaps define some good names as accepted object keys to be expressive by native object key names, and the values of the object keys would be your choice of appropriate data type, i.e.:
```
memoryLeak.destroy({
  isCompleteDestroy: true
});
```
or
```
memoryLeak.destroy({
  isDestroyRefrencesOnly: true
});
```

Think if one sets up the accepted parameter with object with meaningful named object keys to control the `.destroy` API, it would pave the scalability of the `.destroy` API, if the `.destroy` API ever expands.